### PR TITLE
Add dust collection system to 1000mm instructions

### PR DESCRIPTION
**Pull request by rchlfryn**
#### What does this do?
Since the fully loaded release, CS has been receiving a lot of calls about dust control system instructions. This adds a copy of the step to assemble the dust control system from the /upgrade section to the 1000mm instructions.

Screen Shot 2019-12-19 at 11 49 03 AM

### CODE REVIEW

**Review by nevernormal2 — CHANGES_REQUESTED** [review]
Is there a way to use a shared partial instead of copying the instructions? If we have the same copy in 2 places we might forget to update one those places when it changes.

**Comment by rchlfryn** [comment]
I thought about this. Ultimately that is what I wanted to do as well, but the repo isn't set up for that, so I thought this was a quick fix until we can get things more dynamic.

File: 1000mm/step11/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: "Computer Setup"
+step_number: 1
```
Comment (nevernormal2) [inline 1000mm/step11/index.md]: It looks like this was there before, but it seems strange. Is there a reason why the step number is 1 here but 11 in the permalink?

Comment (rchlfryn) [reply 1000mm/step11/index.md]: It is supposed to be an 11.

**Comment by nevernormal2** [comment]
> I thought about this. Ultimately that is what I wanted to do as well, but the repo isn't set up for that, so I thought this was a quick fix until we can get things more dynamic.

I believe if you create a file in `/includes/dust-collection.html` and populate it with all of the shared content, you can then do the following on both the `/1000mm/step10` and the `update/step5` pages:

`{% include dust-collection.html %}`

I tested it locally and it seemed to work.

**Comment by rchlfryn** [comment]
Good call. Okay I added the dust_control file to includes and made a parent directory for the images.

**Review by nevernormal2 — APPROVED** [review]

**Pull request merged**